src/components/Sidebar/cart/CartSection.js

```diff
@@ -23,26 +25,50 @@ export const styles = {
   },
 };
 
-function CartSection({ classes, section }) {
-  let leftHeaderContent;
-  if (section.schedule.length > 1) {
-    leftHeaderContent = section.schedule
-      .map(schedule => getFormattedClassSchedule(schedule))
-      .reduce(
-        (resultStr, formattedStr) => `${resultStr}, ${formattedStr}`,
-      );
-  } else {
-    leftHeaderContent = getFormattedClassSchedule(section.schedule[0]);
+class CartSection extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showDialog: false,
+    };
+    this.toggleDialog = this.toggleDialog.bind(this);
+  }
+
+  toggleDialog() {
+    this.setState(state => ({ showDialog: !state.showDialog }));
+  }
+
+  render() {
+    const { classes, section } = this.props;
+    const { showDialog } = this.state;
+    let leftHeaderContent;
+    if (section.schedule.length > 1) {
+      leftHeaderContent = section.schedule
+        .map(schedule => getFormattedClassSchedule(schedule))
+        .reduce(
+          (resultStr, formattedStr) => `${resultStr}, ${formattedStr}`,
+        );
+    } else {
+      leftHeaderContent = getFormattedClassSchedule(section.schedule[0]);
+    }
+    const rightHeaderContent = `${section.subjectId} ${section.courseId}`;
+    return (
+      <div>
+        <Section
+          onClickFn={this.toggleDialog}
```

Comment:
I know this isn't a part of your ticket but can you refactor this component's prop to be named `onClick` instead of `onClickFn`? `onClick` is a standard used in react (straight from the DOM's `onClick`) and there's no need for our components to have it called anything different, in fact it's confusing.